src/main/java/com/github/princesslana/eriscasper/gateway/commands/util/ConnectionProperties.java

```diff
@@ -0,0 +1,29 @@
+package com.github.princesslana.eriscasper.gateway.commands.util;
```

Comment:
Why is this in its own package?

`util` is often a bit of a meaningless name because it can mean anything. Often it's a dumping spot because there wasn't a better name. If this is the case, wouldn't it be better off near the class that uses it?

Reply:
We can put it as an inner class to Identify.